internal/devserver/ts_devserver.bzl

```diff
@@ -59,16 +73,20 @@ RUNFILES="$PWD/.."
 {main} {serving_arg} \
   -base "$RUNFILES" \
   -packages={workspace}/{package} \
-  -manifest={workspace}/{manifest}
+  -manifest={workspace}/{manifest} \
+  -concat_files_manifest={workspace}/{concat_files_manifest} \
+  -node_modules_manifest={workspace}/{node_modules_manifest}
```

Comment:
couldn't we just give the paths directly as arguments rather than writing to a file? the string can't get longer than max argv, can it? and it makes debugging one step easier

Reply:
The list of node_modules can be huge so having those in a file makes sense. I didn't reduce them to their base path in skylark. The list of concat_files could be changed passed as arguments although the code is simpler if its done the same way and just passed as a file. I like passing them as files as you can always look in bazel-out and see what got written out.